Makefile

```diff
@@ -13,8 +21,12 @@ bash:
 	@$(COMPOSE_DEV) $(BASH)
 
 .PHONY: ci
-ci:
+ci: ## requires build to be run first, can do "env=dev make ci" to run with docker-compose.yml
+ifeq ($(ENV_ARG), dev)
+	@$(BASH_DEV) "bin/rails db:setup db:migrate ci"
+else
 	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
+endif
 
 .PHONY: console
 console:
```

Comment:
should i update these commands as well to default to `docker-compose.test.yml` and use an `ifeq` ?